src/ui/buttons/button.jsx

```diff
@@ -134,7 +134,7 @@ export function Button({ fundingSource, style, multiple, locale, env, fundingEli
 
     // Only apply animation to the paypal button
     const buttonDesign = fundingSource === FUNDING.PAYPAL
-        ? getButtonDesign(personalization)
+        ? getButtonDesign(personalization, logoColor)
```

Comment:
Instead of passing in LogoColor here, can you pass it through props to the component here?
https://github.com/paypal/paypal-checkout-components/blob/master/src/funding/paypal/template.jsx#L166-L177

Reply:
Yeah I think that is possible, let me check that

Reply:
I tried to pass it but it was not possible, since we already created and  passed the props to the component, and in the file you mentioned we are just rendering the component already built:
![Screen Shot 2021-12-09 at 3 29 12 PM](https://user-images.githubusercontent.com/7244087/145470938-be21ce22-8ad9-4c09-8d2c-5bc6a218a257.png)